src/explorer/DeploymentSlotsTreeItem.ts

```diff
@@ -98,7 +98,7 @@ export class DeploymentSlotsTreeItem implements IAzureParentTreeItem {
                     return `The slot name "${value}" is not available.`;
                 }
 
-                const nameAvailability: ResourceNameAvailability = await nodeUtils.getWebSiteClient(node).checkNameAvailability(`${util.extractSiteName(node.treeItem.site)}-${value}`, 'Slot');
+                const nameAvailability: ResourceNameAvailability = await nodeUtils.getWebSiteClient(node).checkNameAvailability(`${value}`, 'Slot');

```

Comment:
This might fix the problem that you described, but it breaks duplicate names. Even though I already have 'erijiz1', I'm not getting an error:
![screen shot 2018-03-01 at 12 18 36 pm](https://user-images.githubusercontent.com/11282622/36867780-0f438314-1d4b-11e8-9f3f-d6bffb265085.png)
![screen shot 2018-03-01 at 12 18 30 pm](https://user-images.githubusercontent.com/11282622/36867785-10a96b74-1d4b-11e8-9add-f942c223a000.png)